android/WALT/app/src/main/java/org/chromium/latency/walt/MainActivity.java

```diff
@@ -91,6 +91,7 @@ protected void onResume() {
         final UsbDevice usbDevice;
         Intent intent = getIntent();
         if (intent != null && intent.getAction().equals(UsbManager.ACTION_USB_DEVICE_ATTACHED)) {
+            setIntent(null); // done with the intent
```

Comment:
Is this the correct order, first setIntent and the get the device out of the saved intent? Feels a bit weird.